poc/vdaf_prio3.py

```diff
@@ -483,6 +510,51 @@ class Prio3HistogramWithLength(Prio3Histogram):
         return Prio3HistogramWithLength
 
 
+class Prio3SumVecWithMultiproof(Prio3SumVec):
+    ID = 0xFFFFFFFF  # To be set by `with_params`
+
+    # Operational parameters.
+    test_vec_name = 'Prio3SumVecWithMultiproof'
+
+    @staticmethod
+    def is_recommended(valid_cls,
```

Comment:
We might need to take other parameters from the underlying FLP circuits/gadgets into considerations? e.g. `Valid.GADGET_CALLS`, `GADGET.DEGREE`? What constrains shall we specify ?